ProjectSettings/QualitySettings.asset

```diff
@@ -193,7 +193,7 @@ QualitySettings:
     name: Ultra
     pixelLightCount: 4
     shadows: 2
-    shadowResolution: 2
+    shadowResolution: 3
```

Comment:
This changes shadow quality settings?

Reply:
Apparently not, I tried changing to see if it would.